### DISCUSSION TITLE
[FATAL] Unsupported key in configuration file: [fallback_resolver]

### DISCUSSION THREAD

**Discussion by Kylvan-8** [DNSCrypt/dnscrypt-proxy#1979] [Q&A] [answered]
Hello since i've updated my dnscrypt-proxy version to 2.2.1 with the script provided on the wiki

I get this error from my logs:

[FATAL] Unsupported key in configuration file: [fallback_resolver]

Nothing have change in the .toml file. Same as before updating.

Can you please explain me what is going on and how i can resolve this issue ?

Best regards and happy new year

**Comment by jedisct1** [comment, marked as answer]
`fallback_resolver` was deprecated in version 2.0.38.

You should replace it with `bootstrap_resolvers`, that takes a list of possible resolvers instead:

https://github.com/DNSCrypt/dnscrypt-proxy/blob/916e84e798989893cb3a2c35402f32b01264dd78/dnscrypt-proxy/example-dnscrypt-proxy.toml#L214-L244

But since you jumped a couple versions, maybe it would be better to start fresh from the current example file.

**Reply by Kylvan-8** [reply]
Thank you for your quick reply jedisct as usual !
